relay-enum-generator.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import * as mkdir from 'mkdirp-sync';
+import * as path from 'path';
+import { TypeKind, buildSchema, graphql, introspectionQuery } from 'graphql';
+
+const relayConfig = require('relay-config').loadConfig();
+
+function findRelayConfigOption(option: string) {
+  return relayConfig[option];
+}
+
+const schemaPath = findRelayConfigOption('schema');
+// TODO: get this from cli
+const writePath = './src';
+// TODO: get this from cli
+const filename = 'enums.ts';
+
+async function parseSchema(rawSchemaContent: string, schemaPath: string) {
+  const schemaType = schemaPath
```

Comment:
you can use `extname` function of `path` module

https://nodejs.org/api/path.html#path_path_extname_path